e2e/connection_test.go

```diff
@@ -24,9 +24,10 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestClientConnectsToNode(t *testing.T) {
+// This test can be invoked only after node registers its identity, thus it is called from TestOwnIdentityRegistrationFlow
+func clientConnectsToNodeTest(t *testing.T) {
```

Comment:
It's not test, should be something as `assert*()`

Reply:
Plus `client registration flow` -> calls `connection flow`. Should be more clear dependencies of flows order

Reply:
Made single flow